CMakeLists.txt

```diff
@@ -108,7 +108,8 @@ get_version ( ${RVS_VERSION} )
 # set(CMAKE_PACKAGING_INSTALL_PREFIX "/opt/rocm")
 set(CPACK_PACKAGE_NAME "rocm-validation-suite")
 set(CPACK_PACKAGE_DESCRIPTION "ROCm Validation Suite")
-set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "The ROCm Validation Suite – The ROCm Validation Suite is a system administrator and cluster manager's tool for detecting and troubleshooting common problems affecting AMD GPUs running in high performance computing environments, enabled using the ROCm software stack on a compatible platform.")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "The ROCm Validation Suite validates the AMD platform using ROCM Runtime.
+ The ROCm Validation Suite is a system administrator and cluster manager's tool for detecting and troubleshooting common problems affecting AMD GPUs running in high performance computing environments, enabled using the ROCm software stack on a compatible platform.")
```

Comment:
using ROCM Runtime  ==> Should be ROCm , as you have done with the whole sentence

Also, kindly change " The ROCm Validation Suite – "  to  "ROCm Validation Suite –" ,  since we are at 'this' package, In my view "the" should not be used.

Reply:
Done